app/controllers/posts_controller.rb

```diff
@@ -42,9 +44,14 @@ def edit
   end
 
 
-  def draft
-    @drafts =  Post.order(created_at: :desc).page(params[:page]).per 7
-  end
+  def draft 
+    @posts=  Post.all
```

Comment:
Don't use the .all api, i told you before.

Reply:
Yes, the draft methods will be corrected by @mithun30052001